### PR TITLE
Adding new secret for registry.redhat.io

### PR DESCRIPTION
**Pull request by thekad**

### CODE REVIEW

**Comment by thekad** [comment]
cc @kargakis

File: projects/azure/README.md [hunk 1]
```diff
@@ -251,6 +251,7 @@ we will not need to have the rest of the secrets in place.
   --from-file=cluster/test-deploy/azure/ssh-privatekey \
   --from-file=cluster/test-deploy/azure/certs.yaml \
   --from-file=cluster/test-deploy/azure/.dockerconfigjson \
+  --from-file=cluster/test-deploy/azure/redhat-io-docker-config.json \
```
Comment (kargakis) [inline projects/azure/README.md]: I think you also need to update `cluster/test-deploy/azure/.gitignore` to ignore the new secret.

Comment (thekad) [reply projects/azure/README.md]: Updated

Comment (thekad) [reply projects/azure/README.md]: I also added some more missing secrets we recently added

**Pull request merged**